packages/core/admin/admin/src/content-manager/components/RelationInput/RelationInput.js

```diff
@@ -325,35 +325,37 @@ const RelationInput = ({
           )
         }
       >
-        <RelationList overflow={overflow}>
-          <VisuallyHidden id={ariaDescriptionId}>{listAriaDescription}</VisuallyHidden>
-          <VisuallyHidden aria-live="assertive">{liveText}</VisuallyHidden>
-          <List
-            height={dynamicListHeight}
-            ref={listRef}
-            outerRef={outerListRef}
-            itemCount={totalNumberOfRelations}
-            itemSize={RELATION_ITEM_HEIGHT + RELATION_GUTTER}
-            itemData={{
-              ariaDescribedBy: ariaDescriptionId,
-              canDrag: canReorder,
-              disabled,
-              handleCancel: onCancel,
-              handleDropItem: onDropItem,
-              handleGrabItem: onGrabItem,
-              iconButtonAriaLabel,
-              labelDisconnectRelation,
-              onRelationDisconnect,
-              publicationStateTranslations,
-              relations,
-              updatePositionOfRelation: handleUpdatePositionOfRelation,
-            }}
-            itemKey={(index, { relations: relationsItems }) => relationsItems[index].id}
-            innerElementType="ol"
-          >
-            {ListItem}
-          </List>
-        </RelationList>
+        {relations.length > 0 ? (
```

Comment:
I find this more readable:

```suggestion
        {relations.length > 0 && (
```

Wdyt?

Reply:
I don't have a huge preference, my only concern is we should avoid doing this in circumstances where the condition does not equate to a boolean e.g  if we just did `relations.length && <Component />` it would potentially render a `0` because of how `&&` works, just something to keep in mind, but also why I normally use a more verbose option – ternary.